redis/commands/core.py

```diff
@@ -1762,14 +1761,12 @@ def getex(
         if exat is not None:
             pieces.append("EXAT")
             if isinstance(exat, datetime.datetime):
-                s = int(exat.microsecond / 1000000)
-                exat = int(time.mktime(exat.timetuple())) + s
+                exat = int(exat.timestamp())
             pieces.append(exat)
         if pxat is not None:
             pieces.append("PXAT")
             if isinstance(pxat, datetime.datetime):
-                ms = int(pxat.microsecond / 1000)
```

Comment:
This can be removed because `timestamp()` returns a fractional result and multiplying with 1000 will therefore already consider ms.